### DISCUSSION TITLE
Takeoff roll is a ball - and not in the good sense (dev and stable)

### DISCUSSION THREAD

**Discussion by robinottawa** [flybywiresim/aircraft#3242] [Q&A] [answered]
For the last few weeks, the a/c has been gyrating wildly just before rotation and past, and I always thought it must be the dev version, but it's not. It does it in both. Seems bad. If it was a real a/c, it would be scraping the wings into the ashphalt every time.

**Comment by omarsmak** [comment, marked as answer]
Try the experimental version with the custom fly by wire. It solves all these problems

**Reply by robinottawa** [reply]
I thought it would be worse than the dev version. I'll give it a try. Thx

**Reply by robinottawa** [reply]
> 
> 
> Try the experimental version with the custom fly by wire. It solves all these problems

It worked perfectly. Why is this here if it's better than the dev version, and the stable verison for that matter?!?  Thx

**Reply by omarsmak** [reply]
Because they have completely implemented their own custom fly by wire system which is complicated and hence no longer relay on Asobo's one. I think once it gets more stable, it will be merged into the dev branch although I think this may happen soon as this experimental is getting better and better with every iteration
